components/CalendarReminder.js

```diff
@@ -97,16 +97,23 @@ export default function CalendarReminder() {
 
   function watering(array, addDay, modulo) {
     return array.map((plant) =>
-      (day + addDay) % modulo === 0 ? plant.name + ", " : ""
+      (day + addDay) % modulo === 0 ? (
+        <li key={plant._id}>{plant.name}</li>
+      ) : null
```

Comment:
When making such a check, you can also make use of the logical and operator to make the code even easier to read:

```suggestion
      (day + addDay) % modulo === 0 && <li key={plant._id}>{plant.name}</li>
```